docs/features.md

```diff
@@ -0,0 +1,48 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.MsBuild addin.
+---
+The [Cake.Issues.MsBuild addin] provides the following features.
+
+# Basic features
+
+* Reads warnings from MSBuild log files.
+* Provides URLs for all code analysis (`CA*`) and StyleCop (`SA*`) warnings.
+* Support for custom URL resolving using the [MsBuildAddRuleUrlResolver] alias.
+
+# Supported log file formats
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.